docs/installation.rst

```diff
@@ -0,0 +1,103 @@
+Installation
+============
+
+The page has information for installing ``openfe``, installing software
+packages that integrate with ``openfe``, and testing that your ``openfe``
+installation is working.
+
+``openfe`` currently only works on POSIX system (macOS and UNIX/Linux). It
+is tested against Python 3.9 and 3.10.
+
+Installing ``openfe``
+---------------------
+
+When you install ``openfe`` through any of the methods described below, you
+will install both the core library and the command line interface.
+
+Installation with ``conda`` (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We recommend installing ``openfe`` with ``conda``, because it provides easy
+installation of other tools, including molecular dynamics tools such as
+OpenMM and ambertools, which are needed by ``openfe``. 
+Conda can be installed either with the `full Anaconda distribution
+<https://www.anaconda.com/products/individual>`_, or with
+the `smaller-footprint miniconda
+<https://docs.conda.io/en/latest/miniconda.html>`_. We also recommend
+`mamba
+<https://mamba.readthedocs.io/en/latest/installation.html#installation>`_ as
+a more perfomant variant of ``conda``.
```

Comment:
if you're having to describe conda, it probably doesn't help to describe the three different variants all at once.

Pick a thing and run with it.  We "recommend" mamba, but then don't actually use this in the example.

I'd just trim this to; use conda, here's miniconda, done